### PR TITLE
remove Pool.active and clean up

### PR DESCRIPTION
**Pull request by iamchrissmith**
remove the pool.active function

add conditions to the plan.active function

clean up spacing

### CODE REVIEW

**Comment by talbaneth** [comment]
This might not be ready for review, but can I ask what is the idea with the new `wild` getter?

**Comment by iamchrissmith** [comment]
> This might not be ready for review, but can I ask what is the idea with the new `wild` getter?

It came up in the red team review as a way we can permissionlessly cage a pool/plan that is misconfigured or misbehaving.  

Tbh I am not liking all these different states for pools, plans, d3ms etc. But thought we could review from this PR and try to combine them

**Comment by iamchrissmith** [comment]
per @talbaneth in compound review: should make sure we document active and disable relationship

File: src/plans/D3MAavePlan.sol [hunk 1]
```diff
@@ -200,6 +207,19 @@ contract D3MAavePlan is ID3MPlan {
         return strategy == address(tack);
     }
 
+    function pause() external override {
+        paused_ = true;
+    }
```
Comment (iamchrissmith) [inline src/plans/D3MAavePlan.sol:212]: needs to be `auth` and add tests

File: src/plans/D3MAavePlan.sol [hunk 2]
```diff
@@ -194,8 +193,13 @@ contract D3MAavePlan is ID3MPlan {
 
     function active() public view override returns (bool) {
         if (bar == 0) return false;
-        (,,,,,,,,,, address strategy,) = pool.getReserveData(address(dai));
-        return strategy == address(tack);
+        (,,,,,,, address adai_, address stableDebt_, address variableDebt_, address strategy,) = pool.getReserveData(address(dai));
+        return (
+                    strategy == address(tack) &&
```
Comment (naszam) [inline src/plans/D3MAavePlan.sol:198]: nit: consider aligning `&`

Comment (iamchrissmith) [reply src/plans/D3MAavePlan.sol]: ae618a4

**Pull request merged**